website/blog/2020-02-31-version-60.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Announcing React Native 0.60
+author: Ryan Turner
+authorTitle: Core Maintainer & React Native Developer
+authorURL: https://twitter.com/turnrye
+authorImageURL: https://avatars0.githubusercontent.com/u/701035?s=460&v=4
+authorTwitter: turnrye
+category: announcements
+---
+
+After months of hard work from hundreds of contributors, the React Native Core team is proud to announce the release of version 0.60. This release handles significant migrations for both Android and iOS platforms, and many issues are resolved too. This blog post covers the highlights of the release. As always though, refer to the changelog for more detailed information. Finally, thank you contributors for helping us make this milestone!
```

Comment:
```suggestion
After months of hard work from hundreds of contributors, the React Native Core team is proud to announce the release of version 0.60. This release handles significant migrations for both Android and iOS platforms, and many issues are resolved too. This blog post covers the highlights of the release. As always though, refer to the changelog for more detailed information. Finally, thank you, contributors, for helping us make this milestone!
```

Reply:
While "contributors" is used as an appositive here, since "you" is imprecise (for instance the reader may not be a contributor), there need not be commas around it.